debian/usr.bin.ubuntu-core-launcher

```diff
@@ -19,6 +19,10 @@
 
     /usr/bin/ubuntu-core-launcher r,
 
+    # for snap run
+    /usr/bin/snap mr,
+    /lib/@{multiarch}/linux-vdso.so* mr,
```

Comment:
vdso is not a real shared library, the kernel maps and passes the location via the auxiliary vector. Are you sure this is required?